src/hooks/resourceHooks/useLocalGokbPkg.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+import { useOkapiKy } from '@folio/stripes/core';
+
+import { PACKAGES_ENDPOINT } from '../../constants';
+
+const useLocalGokbPkg = ({ reference }) => {
+  const ky = useOkapiKy();
+  const remoteId = reference ? reference.split(':')[0] : null;
+  const { data } = useQuery(
+    ['PKG', 'fetchLocalPkg', remoteId],
+    () => ky.get(`${PACKAGES_ENDPOINT}?filters=identifiers.identifier.value==${remoteId}&&identifiers.identifier.ns.value==gokb_id&&identifiers.identifier.status.value==approved`).json(),
```

Comment:
We have a query building function, we should use it instead of doing this directly (again though I think that this probably belongs in the usePackage, centralising the query params somwhat